### DISCUSSION TITLE
绳网问题

### DISCUSSION THREAD

**Discussion by Bahajsjsh** [share121/inter-knot#3283] [其他]
![Screenshot_20240820_153216](https://github.com/user-attachments/assets/2c7cb756-5d43-4ee9-9b55-96ec7164fe69)
这是什么问题求解答我打开了绳网创建了账号发了评论但就是显示不出来

**Comment by DJLFi** [comment]
清网站缓存就行

**Reply by Bahajsjsh** [reply]
清理过了没有

**Reply by Bahajsjsh** [reply]
这个事情我也在B站那边发过有人说需要搭建梯子我也不知道是不是需要搭建梯子

**Reply by HitoriGotoh-Y** [reply]
应该是不用的，我没挂梯子但是也能正常用
